Source/Scene/Camera.js

```diff
@@ -3230,9 +3240,35 @@ define([
         Matrix4.clone(camera._transform, result.transform);
         result._transformChanged = true;
 
+        if (defined(camera.cameraLimiter)) {
+            CameraLimiter.clone(camera.cameraLimiter, result.cameraLimiter);
```

Comment:
I don't think this is covered by the tests. You should be able to just add a line to the `clone` test.

Reply:
done.